@here/harp-mapview/lib/Utils.ts

```diff
@@ -603,6 +687,46 @@ export namespace MapViewUtils {
         return { tilt, azimuth };
     }
 
+    /**
+     * Gets the tilt angle (in radians) of the object relative to the coordinates of `location`.
+     *
+     * Note: this method can be used to get the direction that an object points to, when `location`
+     * is the target of that object, by adding PI to it. Otherwise it only returns the tilt angle
+     * (in radians) of `object` in the tangent space of `location`.
+     *
+     * @param mapView The [[MapView]] instance to consider.
+     * @param object The object to get the coordinates from.
+     * @param location The reference point.
+     */
+    export function extractTiltAngleFromLocation(
+        mapView: MapView,
```

Comment:
don't pass whole MapView but only what you need

Reply:
As said before, I have changed/fixed only mine methods to use the minimum required input set, but rest of the methods left untouched.